packages/@sanity/components/src/dialogs/DefaultDialog.js

```diff
@@ -114,6 +114,7 @@ export default class DefaultDialog extends React.PureComponent {
         kind={action.kind}
         inverted={action.inverted}
         autoFocus={action.autoFocus}
+        icon={action.icon}
```

Comment:
Should be added to the proptypes-shape for the `action`-prop in this component.

Reply:
👍